### ISSUE TITLE
Error while fine tuning M2M 100 model for a lang pair.

### ISSUE THREAD

**Issue by ShihabYasin** [pytorch/fairseq#3221]
### Getting this error while trying to fine tuning M2M 100 model for en-fr pair.

![a](https://user-images.githubusercontent.com/14265415/107241068-d3e92080-6a54-11eb-86e8-8afdd451ce8b.png)

Command was like:

`
lang_list='Models/lang_list.txt' # attached within this issue 
lang_pairs="en-fr,fr-en"

CUDA_VISIBLE_DEVICES=0 fairseq-train data_bin --lang-pairs "$lang_pairs"  --finetune-from-model Models/418M_last_checkpoint.pt --fixed-dictionary Models/model_dict.128k.txt --encoder-normalize-before --decoder-normalize-before --arch transformer --layernorm-embedding --task translation_multi_simple_epoch --sampling-method "temperature" --sampling-temperature 1.5 --encoder-langtok "src" --decoder-langtok --lang-dict "$lang_list" --criterion label_smoothed_cross_entropy --label-smoothing 0.2 --optimizer adam --adam-eps 1e-06 --adam-betas '(0.9, 0.98)' --lr-scheduler inverse_sqrt --lr 3e-05 --warmup-updates 2500 --max-update 40000 --dropout 0.3 --attention-dropout 0.1 --weight-decay 0.0 --max-tokens 1024 --update-freq 2 --save-interval 1 --save-interval-updates 5000 --keep-interval-updates 10 --no-epoch-checkpoints --seed 222 --log-format simple --log-interval 2 --decoder-langtok --encoder-langtok src --fp16 --gen-subset test --dataset-impl mmap --task translation_multi_simple_epoch --distributed-world-size 1 --distributed-no-spawn
`

[lang_list.txt](https://github.com/pytorch/fairseq/files/5944618/lang_list.txt)

### Anybody had the same issue with a possible solution suggestion?

**Comment by ShihabYasin** [comment]
Is there any update after assigning to @shruti-bh by @alexeib ?

**Comment by shruti-bh** [comment]
what's the directory structure of "data_bin"?

**Comment by ShihabYasin** [comment]
data_bin is at project root.

**Comment by gonkillua0501** [comment]
> ### Getting this error while trying to fine tuning M2M 100 model for en-fr pair.
> ![a](https://user-images.githubusercontent.com/14265415/107241068-d3e92080-6a54-11eb-86e8-8afdd451ce8b.png)
> 
> Command was like:
> 
> `
> lang_list='Models/lang_list.txt' # attached within this issue
> lang_pairs="en-fr,fr-en"
> 
> CUDA_VISIBLE_DEVICES=0 fairseq-train data_bin --lang-pairs "$lang_pairs" --finetune-from-model Models/418M_last_checkpoint.pt --fixed-dictionary Models/model_dict.128k.txt --encoder-normalize-before --decoder-normalize-before --arch transformer --layernorm-embedding --task translation_multi_simple_epoch --sampling-method "temperature" --sampling-temperature 1.5 --encoder-langtok "src" --decoder-langtok --lang-dict "$lang_list" --criterion label_smoothed_cross_entropy --label-smoothing 0.2 --optimizer adam --adam-eps 1e-06 --adam-betas '(0.9, 0.98)' --lr-scheduler inverse_sqrt --lr 3e-05 --warmup-updates 2500 --max-update 40000 --dropout 0.3 --attention-dropout 0.1 --weight-decay 0.0 --max-tokens 1024 --update-freq 2 --save-interval 1 --save-interval-updates 5000 --keep-interval-updates 10 --no-epoch-checkpoints --seed 222 --log-format simple --log-interval 2 --decoder-langtok --encoder-langtok src --fp16 --gen-subset test --dataset-impl mmap --task translation_multi_simple_epoch --distributed-world-size 1 --distributed-no-spawn
> `
> 
> [lang_list.txt](https://github.com/pytorch/fairseq/files/5944618/lang_list.txt)
> 
> ### Anybody had the same issue with a possible solution suggestion?

You may need to make sure inside data_bin has train.* and valid.*

**Comment by maroxtn** [comment]
Try adding validation data, I think fairseq is failing to find it
